source/api/annex/notes/scheduled-changes.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Scheduled Changes"
+layout: spec
+tags: [annex, presentation-api, image-api]
+---
+
+# Introduction
+
+This document records scheduled changes that are intended to be made in the future, but must wait for new major releases as they're backwards incompatible.
+
+# Image API
+
+* Require support for the `square` region at compliance level 1.
+* Clarify that height and width are for calculating aspect ratio only, as there is no 'source' image.
+
```

Comment:
I wonder if it's worth adding links to the relevant Github issues for each of the bullets? I recognize that this is a deeper view into our process than we'd probably want to publish most of the time, but in this case there's a conversation/rationale that supports each item and I think that should be surfaced somehow.